libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -368,41 +368,43 @@ void AP_MotorsMatrix::output_armed_stabilizing()
     // calculate how close the motors can come to the desired throttle
     rpy_high *= rpy_scale;
     rpy_low *= rpy_scale;
-    throttle_thrust_best_rpy = -rpy_low;
-    float thr_adj = throttle_thrust - throttle_thrust_best_rpy;
-    if (rpy_scale < 1.0f) {
-        // Full range is being used by roll, pitch, and yaw.
-        limit.roll = true;
-        limit.pitch = true;
-        limit.yaw = true;
-        if (thr_adj > 0.0f) {
+    const float final_throttle = [&]() {
```

Comment:
hahahaaha I am really curious about what the other will say about this.
 In this case, I don't think this will bring much improvements as the compiler should already handle this simple variable.

Reply:
yes, i dont expect any improvements. possibly no changes at all. but the compiler works in mysterious ways...